libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -139,6 +139,12 @@ + (void)initialize {
         // For dev support
         method_exchangeImplementations(class_getInstanceMethod([UIWindow class], @selector(motionEnded:withEvent:)), class_getInstanceMethod([UIWindow class], @selector(sfsdk_motionEnded:withEvent:)));
 
+        // Pasteboard
```

Comment:
@bbirman I believe that swizzling guidance is to call during `+load`. Summary here: https://nshipster.com/method-swizzling/ (while not the definitive answer, nshipster has been a good summary reference).

`+initialize` _should_ work within the context of the SalesforceSDKManager (as it is not a category) but it feels like moving to recommended patterns may be better, esp at this core level.

Reply:
@bbirman any thoughts on the above?

Reply:
It’s updated in #3099